src/gwkokab/models/hybrids/_bp2pmultispinfull.py

```
@@ -0,0 +1,688 @@
+        )
+
+        _Z_q = jnp.interp(m1, self._m1s, self._Z_q_given_m1, left=1.0, right=1.0)
+        safe_Z_q = jnp.where(_Z_q <= 0, 1.0, _Z_q)
+        log_Z_q = jnp.where(_Z_q <= 0, 0.0, jnp.log(safe_Z_q))
+        log_prob_q = self._log_prob_q_unnorm(m1, m2 / m1) - log_Z_q
+
+        return jnp.log(m1) + log_prob_m1_a1_a2_t1_t2 + log_prob_q - self._logZ
+
+
+def BrokenPowerlawTwoPeakMultiSpinMultiTiltFull(
+    use_eccentricity: bool = False,
```

Ruff (0.14.2)

Boolean-typed positional argument in function definition

(FBT001)